Nikita.py

```
@@ -619,8 +619,8 @@ def start_all(wait=False):
         g.threads.parser                                    =   p.parser("lgp")
         g.threads.parser.start(); # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
         # ~~~~~~~ создаю и запускаю потоки парсеров для нового формата ЖР ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-        #g.threads.parser_new                                =   p.parser("lgd")
-        #g.threads.parser_new.start(); # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        g.threads.parser_new                                =   p.parser("lgd")
+        g.threads.parser_new.start(); # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Ruff (0.15.18)

[error] Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)